mcu/atmega-hal/src/port.rs

```diff
@@ -1,4 +1,9 @@
-#[cfg(any(feature = "atmega48p", feature = "atmega168", feature = "atmega328p"))]
+#[cfg(any(
+    feature = "atmega8u2",
```

Comment:
I don't think this is the right place.  ATmega8U2 has a pin PC7 which is not present on the other chips listed here.  The `impl_port_traditional!` macro sadly doesn not support conditional compilation for single pins yet so you'll have to duplicate the block for the time being (or check if one of the other impls looks better).

Reply:
Oh good catch, I missed this.  Thanks!